homeassistant/components/tado/services.py

```diff
@@ -40,7 +40,12 @@ async def add_meter_reading(call: ServiceCall) -> None:
         reading: int = call.data[CONF_READING]
         _LOGGER.debug("Add meter reading %s", reading)
 
-        tadoconnector = hass.data[DOMAIN][entry_id][DATA]
+        entry = hass.config_entries.async_get_entry(entry_id)
+        if entry is None:
+            raise ConfigEntryNotReady("Config entry not found")
```

Comment:
We can't raise this here, this should lead to a ServiceValidation error (as the user would input an non-existing config entry).

Reply:
Unresolving, as this one was not addressed (and another place was adjusted instead).